Gopkg.toml

```diff
@@ -67,7 +67,7 @@
 
 [[constraint]]
   name = "gopkg.in/natefinch/lumberjack.v2"
-  version = "2.1.0"
+  version = "2.0.0"
```

Comment:
I was going with the release list here: https://github.com/natefinch/lumberjack/tags    I was not sure which was right, since their versioning semantic is different for the most recent release.  If it should be 2.1.0, let's definitely change back.

Reply:
In the toml file I would specify something like `version = "^2.0.0"` to pin to any 2.x.y release.